swap_server_client.go

```diff
@@ -82,6 +107,7 @@ func newSwapServerClient(cfg *ClientConfig, lsatStore lsat.Store) (
 	return &grpcSwapServerClient{
 		conn:   serverConn,
 		server: server,
+		quit:   make(chan struct{}),
```

Comment:
Isn't quit already handled through the contexts that are passed in on every method?

Reply:
indeed, can just rely on ctx